source/blog/2018-07-16-ember-3-3-released.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ember 3.3 Released
+author: Melanie Sumner, Brendan McLoughlin
+tags: Releases, 2018, 3, 3.3
+responsive: true
+---
+
+Today the Ember project is releasing version 3.3.0 of Ember.js, Ember Data, and Ember CLI. We know, 3.2 seems like it just came out yesterday! However, we are really continuing 3.3 today because Ember.js had already completed a beta cycle, so we're catching ember-data & ember-cli up with Ember.
```

Comment:
The wording the last sentence is tough to follow. How about, instead of the last two sentences, something like "While ember-data and ember-cli only released recently, Ember.js released 3.2 over six weeks ago. Releasing 3.3 ensures that Ember.js does not have an excessively long time between releases."